front/pages/api/v1/w/[wId]/vaults/[vId]/apps/index.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint jsdoc/no-missing-syntax: 0 */ //
+// Disabling jsdoc rule, as we're not yet documentating dust apps endpoints under vaults.
+// We still document the legacy endpoint, which does the same thing.
+// Note: for now, an API key only has access to the global vault.
+import type { AppType, WithAPIErrorResponse } from "@dust-tt/types";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { withPublicAPIAuthentication } from "@app/lib/api/wrappers";
+import type { Authenticator } from "@app/lib/auth";
+import { AppResource } from "@app/lib/resources/app_resource";
+import { VaultResource } from "@app/lib/resources/vault_resource";
+import { apiError } from "@app/logger/withlogging";
+
+export type GetAppsResponseBody = {
+  apps: AppType[];
+};
+
+async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<WithAPIErrorResponse<GetAppsResponseBody>>,
+  auth: Authenticator
+): Promise<void> {
+  const owner = auth.workspace();
+  if (!owner) {
+    return apiError(req, res, {
+      status_code: 404,
+      api_error: {
+        type: "workspace_not_found",
+        message: "The workspace you're trying to access was not found",
+      },
+    });
+  }
```

Comment:
Not needed anymore with the new wrapper!

Reply:
nice !